profiles/base/package.use.force

```diff
@@ -38,6 +38,7 @@ media-tv/kodi system-ffmpeg
 # Sam Jorna <wraeth@gentoo.org> (15 Dec 2016)
 # Known to build unbootable EFI binaries on some systems when built with
 # UDK. Forcing use of gnu-efi until it's more stable.
+# This will not be useful anymore as gnuefi use has been deleted.
```

Comment:
Do you mean the flag? Not sure whom the comment is intended to, or how it helps.

Reply:
Yes, the use flag has been deleted in the latest version of rEFInd. I added the comment for we know we  can remove this package.use.force when version 0.11.4 of rEFInd is stabilized. Shall I update the comment to put all this information?

Reply:
No need to change this at all. When all versions having this flag are removed, pkgcheck will tell us it's obsolete.